src/audiences/edit/components/estimate.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { getEstimate } from '../data';
+import { Sparklines, SparklinesLine } from 'react-sparklines';
```

Comment:
I guess this isn't documented in our coding standards properly, but general best practice is to have imports alphabetical by package name, and grouped by:
* Package imports
* Relative imports
* Side-effect imports

```suggestion
import React, { useState, useEffect } from 'react';
import { Sparklines, SparklinesLine } from 'react-sparklines';
import styled from 'styled-components';

import { getEstimate } from '../data';
```